debugging/book-library/script.js

```diff
@@ -77,24 +82,24 @@ function render() {
     cell4.appendChild(changeBut);
     let readStatus = "";
     if (myLibrary[i].check == false) {
-      readStatus = "Yes";
-    } else {
       readStatus = "No";
+    } else {
+      readStatus = "Yes";
     }
     changeBut.innerHTML = readStatus;
 
     changeBut.addEventListener("click", function () {
-      myLibrary[i].check = !myLibrary[i].check;
+      myLibrary[i].check = myLibrary[i].check;
```

Comment:
You have removed ! sign which was negating value of myLibrary[i].check. What your code is doing after your change? Is it expected? Try to create a table for yourself: what will happen to myLibrary[i].check value if it was 'true' initially? If it was 'false'?

Reply:
Thanks for the feedback. I mistakenly removed the part that changes the book's read/unread status. I’ll fix it by adding that part back so the button will work correctly and switch between "Yes" and "No."